### DISCUSSION TITLE
using a spec first approach to openapi instead of autogenerating the spec

### DISCUSSION THREAD

**Discussion by RonnyPfannschmidt** [fastapi/fastapi#6448] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
doesn't apply  here
```

### Description

when developing an api we start with openapi specs, and shared specs/includes to ensure compatibility and coherence across multiple apis

this however requires the web frameworks to take the api spec as input and source of truth instead of the other way around 
i haven't found any examples of fastapi or other modern frameworks supporting such a approach (the sole exception being connexion)

### Operating System

Linux

### Operating System Details

--

### FastAPI Version

--

### Python Version

3.8+

**Comment by raphaelauv** [comment, marked as answer]
There is at least this one python project doing that ->

https://github.com/zalando/connexion

fastAPI is not at all build for that.

You can probably close this issue and continue on the _connexion_ project.

Thanks

**Comment by RonnyPfannschmidt** [comment]
@tiangolo this one is a false positive
